src/backend/server/api/models/shortlist.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+import uuid
+
+
+def default_settings():
+    settings = (
+        {
+            "name": "My List",
+            "color": "red",
+            "icon": {"value": "unknown", "color": "green"},
+        },
+    )
+    return settings
+
+
+class Shortlist(models.Model):
+    shortlist_id = models.UUIDField(
+        primary_key=True, default=uuid.uuid4, editable=False
+    )
+    user_id = models.ForeignKey("authentication.User", on_delete=models.CASCADE)
+    school_ids = ArrayField(
+        models.CharField(max_length=32, null=True),
+        size=4,
+        blank=True,
+        default=list,
+    )
+    shortlist_name = models.CharField(max_length=50)
```

Comment:
Could you explain the use case for shortlist_name versus settings.name?

Reply:
we will keep shortlist_name.